ChangeLog.d/secp256k1-removal.txt

```diff
@@ -0,0 +1,2 @@
+Removals
+   * Support for secp192k1, secp192r1, secp224r1 EC curves is removed from TLS.
```

Comment:
And secp224k1. That happened separately as part of making `USE_PSA_CRYPTO` mandatory, but wasn't announced in a changelog  entry.

Reply:
Not your fault: it should have been done before. There might have been a debate as to whether announce it in Mbed TLS, but it hasn't been announced in crypto either, [I filed an issue for that](https://github.com/Mbed-TLS/TF-PSA-Crypto/issues/412).